python/desc/sims_truthcatalog/sne_truth.py

```diff
@@ -87,33 +91,128 @@ def __getattr__(self, attr):
         # Pass any attribute access requests to the SNObject instance.
         return getattr(self.sn_obj, attr)
 
+    
+def _process_chunk(db_lock, outfile, sne_db_file, sne_db_query, opsim_df,
+                   fp_radius, process_num, max_rows, dry_run, verbose):
+    '''
+    Compute and write out variability for a manageable chunk of SNe
+    '''
+
+    if verbose or dry_run:
+        print("_process_chunk called for process #{} ".format(process_num))
+        print("query_string: \n{}".format(sne_db_query))
+
+    # table_name should perhaps be yet another argument
+    table_name = 'sn_variability_truth'
+    
+    if dry_run:
+        return
+
+    # Get our batch from sne_db_file
+    with sqlite3.connect(sne_db_file) as conn:
+        sne_df = pd.read_sql(sne_db_query, conn)
+    if (verbose):
```

Comment:
It's preferred to omit the parentheses for simple conditions like this.

Reply:
Ok. Old C habits die hard.